Sources/Commands/SwiftPackageTool.swift

```diff
@@ -30,44 +30,44 @@ import func POSIX.chdir
 extension PackageToolOptions: XcodeprojOptions {}
 
 private enum Mode: Argument, Equatable, CustomStringConvertible {
-    case Init(InitMode)
+    case initPackage
     case doctor
-    case showDependencies(ShowDependenciesMode)
+    case showDependencies
     case fetch
     case update
     case usage
     case version
-    case generateXcodeproj(String?)
-    case dumpPackage(String?)
+    case generateXcodeproj
+    case dumpPackage
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
         case "init", "initialize":
-            self = try .Init(InitMode(pop()))
+            self = .initPackage
         case "doctor":
             self = .doctor
-        case "show-dependencies", "-D":
-            self = try .showDependencies(ShowDependenciesMode(pop()))
+        case "show-dependencies", "show-deps":
+            self = .showDependencies
         case "fetch":
             self = .fetch
         case "update":
             self = .update
         case "help", "usage", "--help", "-h":
             self = .usage
-        case "version":
+        case "version", "--version":
```

Comment:
It is to conform with the other commands.  There is precedent in commands like git (which is not a perfect role model of command line interface design, for course).  If we leave out the dash-less version, we're saying there's no command to print the version, just an option.